src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -10,8 +10,8 @@
 interface ElementEventBindingData {
     addDOMEvent?: (elementSelectionBehaviour?: ElementSelectionBehaviour) => void;
     eventID: EventID;
-    removeDOMEvent?: (elementSelectionBehaviour?: ElementSelectionBehaviour) => void;
     name: string;
+    removeDOMEvent?: (elementSelectionBehaviour?: ElementSelectionBehaviour) => void;
```

Comment:
He didn't replace it, but he moved it so that all fields are in alphabetical order. :)